README.md

```diff
@@ -42,51 +42,59 @@ Your code interview will be evaluated based on your repo, so make sure all files
+### Python Task 3: Data Visualization
 
-### Task 4: Spatial data processing
+Write a bash/python process to create a line plot to show the total service request counts by `created_time` that breaks down the data by `complaint_type` if that column is present. Save as many `.png` files of your plots as you think is necessary to show us that your code works. 
+If you would like to go back and download different tables to visualize, that's totally fine. Chose whichever pulls you think make it easiest to show us your work. 
 
-At Data Engineering, we enhance datasets with geospatial attributes, such as point locations and administrative boundaries. To help us better understand the data from **Task 1**, we would like you to join the initial raw data to the **[2020 NTA (Neighborhood Tabulation Area) boundaries](https://www1.nyc.gov/site/planning/data-maps/open-data/census-download-metadata.page)** and create a choropleth map of 7 day total count by NTA of a specific `complaint_type` of your choice.
+### Python Task 4: Spatial data processing
 
-Depending on how you generate the map, you can store the map as a `.png` or `.html` under the `data` folder.
+At Data Engineering, we enhance datasets with geospatial attributes, such as point locations and administrative boundaries. To help us better understand the data from **Python Task 1**, we would like you to join the initial raw data to the some NYC administraive boundary of your chosing. Then create a choropleth map of count of complaints of a specific `complaint_type` of your choice.
 
-### Task 5: SQL
+Depending on how you generate the map, you can store the map as a `.png` or `.html` under the `data` folder. Please add a title that accurately tells the reader what they are looking at.
 
-We ❤️ SQL! At Data Engineering, we deal with databases a lot and we write a lot of fast and simple ETL pipelines using SQL. In this task, you will:
 
-- Load the `data/raw.csv` into a database of your choice and name the table `sample_311`. Make sure this process is captured in a script.
-- Perform the same aggregation in **Task 2** in SQL and store the results in a table (same name as the corresponding csv file).
+### SQL/Docker Task 1: Build container and load 
+
+We ❤️ SQL and docker! At Data Engineering, we work with database containers a lot and we write a lot of fast and simple ETL pipelines using SQL. In this task, you will:
 
-> Note: Depending on your preference, you can use or [Postgres](https://www.postgresql.org/), which is prefered; however, if you are familiar with [SQLite](https://docs.python.org/3/library/sqlite3.html) (much easier to set up and use), you can use that too.
+- Set up POSTGIS container using an image
+- Load the `data/raw.csv` into the database and name the table `sample_311`. Make sure this process is captured in a script.
```

Comment:
You didn't tell them how to name files up top, so `data/raw.csv` is out of place here.  Naming is hard.  I'm okay with providing some guidance to make things easier to explain downstream.

Reply:
Oh yea I can change that